Makefile

```diff
@@ -49,21 +49,12 @@ clippy:
 
 
 ci: fmt clippy test
-	git diff --exit-code Cargo.lock
```

Comment:
The muta doesn't have cargo. Lock, so this is deleted, right? @zeroqn

Reply:
review commit

Reply:
ok test

Reply:
ok-test

Reply:
ok-to-test

Reply:
ok-to-test

Reply:
Yes